src/Data/CartItem/Cart.ts

```diff
@@ -72,6 +76,8 @@ export async function checkoutCart(): Promise<DataItem> {
 async function getTitle() {
   const name = await load(() => currentUser()?.name())
 
+  if (name === undefined) return
```

Comment:
If I'm not mistaken this would create an Opportunity with `undefined` for the `keyword`.

This looks more like an exceptional case: Either throw an error or provide a fallback keyword.